### ISSUE TITLE
make (browserify) requires case sensitive to prevent errors on linux server

### ISSUE THREAD

**Issue by AtiX** [ad-si/brickify#221]

**Comment by adius** [comment]
What exactly is the problem?

**Comment by Steditor** [comment]
Wrong casing in `require`s is ignored on Windows and Mac, but leads to a server crash on Linux. There are npm modules that check the file names in javascript, but I haven't found one for coffee-script yet.

**Comment by adius** [comment]
Yeah I know, but if we use the correct spelling there is no problem, right?
